### DISCUSSION TITLE
Will Recommended Client Version be getting updated?

### DISCUSSION THREAD

**Discussion by awittner** [Esri/arcgis-cookbook#377] [General]
Recommended Client version to Cookbook version is listed https://esri.github.io/arcgis-cookbook/versions.html and the client version listed for use with 4.2.0 is older than the client version I am using. Cinc-client 17.10.0 reached End Of Life on 5/1/23, yet the recommended version is 16+. Cinc Client 18.3 was released 9/21/23. Can you confirm support for more current client versions? IT would prefer use of a supported client version, inclusive of security updates. 

![image](https://github.com/Esri/arcgis-cookbook/assets/6589458/da3c9300-35f6-4192-ba01-9136a664483b)

**Comment by cameronkroeker** [comment]
Hi @awittner,

Chef/Cinc client versions 17 and 18 are supported (Cookbooks v4.2.0 support greater than or equal to 14. In the table it shows as `Supported Chef Client Versions >= 14.0`). Meaning there isn't anything within the v4.2.0 cookbooks that will block the use of Chef/Cinc client versions 17 and 18.

The recommended Chef/Cinc client versions are what we have tested/certified the cookbooks against at the time of release. I think this can be treated more as a "suggested" version to use because it has our highest confidence in knowing it will work. 

I would say Chef/Cinc client versions 17 and 18 should be fine to use in most cases, however, if there are any issue(s) that are found in 17 and/or 18 but not 16 then that's when the recommended version would be highlighted as the version to use.

> Note: Chef/Cinc client 17 may give several depreciation warning messages at the end of the run but shouldn't have any impact on the run being successful or not.

Thanks,
Cameron K.
